share/normalize/links.py

```diff
@@ -275,9 +276,18 @@ def execute(self, obj):
 
 
 class DateParserLink(AbstractLink):
+    LOWER_BOUND = pendulum.create(1200, 1, 1)
+    UPPER_BOUND = pendulum.now().add(years=100)
+    DEFAULT = pendulum.create(2016, 1, 1)
+
     def execute(self, obj):
         if obj:
-            return arrow.get(obj).to('UTC').isoformat()
+            date = dateutil.parser.parse(obj, default=self.DEFAULT)
```

Comment:
Force the date into UTC here. If there isn't a timezone assume it is already UTC